qiskit/pulse/library/waveform.py

```diff
@@ -26,24 +26,30 @@ class Waveform(Pulse):
     duration of the backend cycle-time, dt.
     """
 
+    limit_amplitude = True
+
     def __init__(
         self,
         samples: Union[np.ndarray, List[complex]],
         name: Optional[str] = None,
+        limit_amplitude: Optional[bool] = None,
```

Comment:
I would prefer this kwarg be added after `epsilon`. While, I agree the current order makes sense if we were to start from scratch if existing code is misusing the `epsilon` keyword argument as a positional argument this change would break said code.